core/domain/event_services.py

```diff
@@ -124,24 +161,34 @@ def _handle_event(
 class ExplorationActualStartEventHandler(BaseEventHandler):
     """Event handler for recording exploration actual start events."""
 
-    EVENT_TYPE = feconf.EVENT_TYPE_ACTUAL_START_EXPLORATION
+    EVENT_TYPE: str = feconf.EVENT_TYPE_ACTUAL_START_EXPLORATION
```

Comment:
same as above

Reply:
EVENT_TYPE is already defined in BaseEventHandler and we are just redefining it in all child classes. Also, if we use Final here then MyPy throws an error. Because Final is only used with Constants but here we are redefining it in child classes.